### PR TITLE
rpc: abci_info fixes

### PR DESCRIPTION
**Pull request by princesinha19**
Added `last_block_height` and `last_block_app_hash` in abci_info of kvstore as it is necessary or important information. Also, removed omitempty from `ResponseInfo` interface.

Fixes: #4117 

* [x] Updated all relevant documentation in docs
* [x] Updated all code comments where relevant
* [x] Updated CHANGELOG_PENDING.md

### CODE REVIEW

File: abci/types/types.pb.go [hunk 1]
```diff
@@ -1235,11 +1235,11 @@ func (m *ResponseFlush) XXX_DiscardUnknown() {
 var xxx_messageInfo_ResponseFlush proto.InternalMessageInfo
 
 type ResponseInfo struct {
-	Data                 string   `protobuf:"bytes,1,opt,name=data,proto3" json:"data,omitempty"`
```
Comment (melekes) [inline abci/types/types.pb.go]: This file is auto generated by protoc, so your changes will be overwritten next time.

Solution we came up with a while ago is to use custom marshallers - https://github.com/tendermint/tendermint/issues/3882. Is this not working?

File: abci/types/types.pb.go [hunk 2]
```diff
@@ -1235,11 +1235,11 @@ func (m *ResponseFlush) XXX_DiscardUnknown() {
...
-	Version              string   `protobuf:"bytes,2,opt,name=version,proto3" json:"version,omitempty"`
-	AppVersion           uint64   `protobuf:"varint,3,opt,name=app_version,json=appVersion,proto3" json:"app_version,omitempty"`
-	LastBlockHeight      int64    `protobuf:"varint,4,opt,name=last_block_height,json=lastBlockHeight,proto3" json:"last_block_height,omitempty"`
-	LastBlockAppHash     []byte   `protobuf:"bytes,5,opt,name=last_block_app_hash,json=lastBlockAppHash,proto3" json:"last_block_app_hash,omitempty"`
+	Data                 string   `protobuf:"bytes,1,opt,name=data,proto3" json:"data"`
```
Comment (marbar3778) [inline abci/types/types.pb.go]: these files are autogenerated, we shouldn't edit them or were these not edited but regenerated?

File: abci/types/types.pb.go (hunk 1 above)
Comment (princesinha19) [reply abci/types/types.pb.go]: Yes, you are correct @melekes. I have added `omitempty` again.

File: CHANGELOG_PENDING.md [hunk 3]
```diff
@@ -55,6 +55,8 @@ program](https://hackerone.com/tendermint).
     ```
     - ID is unique for each request;
     - Request.ID is now optional. Notification is a Request without an ID. Previously ID="" or ID=0 were considered as notifications.
+  - [rpc] \#4177 Added `last_block_height` and `last_block_app_hash` in abci_info of kvstore example. Also removed omitempty from `ResponseInfo` interface (@princesinha19)
```
Comment (melekes) [inline CHANGELOG_PENDING.md]:
```suggestion
  - [abci] \#4177 kvstore: Return `LastBlockHeight` and `LastBlockAppHash` in `Info` (@princesinha19)
```

Comment (princesinha19) [reply CHANGELOG_PENDING.md]: Done @melekes.

File: CHANGELOG_PENDING.md [hunk 4]
```diff
@@ -55,6 +55,8 @@ program](https://hackerone.com/tendermint).
     ```
     - ID is unique for each request;
     - Request.ID is now optional. Notification is a Request without an ID. Previously ID="" or ID=0 were considered as notifications.
+    - [abci] \#4177 kvstore: Return `LastBlockHeight` and `LastBlockAppHash` in `Info` (@princesinha19)
```
Comment (melekes) [inline CHANGELOG_PENDING.md]: please move to `### IMPROVEMENTS:` section

Comment (princesinha19) [reply CHANGELOG_PENDING.md]: Done

**Pull request merged**